cms/static/js/views/settings/grading.js

```diff
@@ -298,10 +303,14 @@ var GradingView = ValidatingView.extend({
         this.descendingCutoffs.push({designation: this.GRADES[gradeLength], cutoff: failBarWidth});
         this.descendingCutoffs[gradeLength - 1]['cutoff'] = Math.round(targetWidth);
 
-        var $newGradeBar = this.gradeCutoffTemplate({ descriptor : this.GRADES[gradeLength],
-            width : targetWidth, removable : true });
+        var newGradeHtmlString = this.gradeCutoffTemplate({
+            descriptor : this.GRADES[gradeLength],
+            width : targetWidth,
+            contenteditable: true,
+            removable : true }).toString();
         var gradeDom = this.$el.find('.grades');
-        gradeDom.children().last().before($newGradeBar);
+
+        gradeDom.children().last().before(newGradeHtmlString);
```

Comment:
@andy-armstrong I need your input on this.  We currently have `HtmlUtils.append` and `HtmlUtils.prepend`.  JQuery has approximately 15 functions of this type.  It feels like it might be a lot of jumping through hoops to refactor down to only 2, or even 4 functions.

Wouldn't your `HtmlUtils.element()` function be a more scalable solution?  Wouldn't this be the one function that rules them all? :)  We could have:
```
gradeDom.children().last().before(HtmlUtils.element(newGradeHtml));
``` 
Note that the linter could also make sure that this method is only used in the context of a JQuery function call, so it isn't used prematurely, losing the HTML snippet-ness before its time.

If we went down this route, would it still be useful to even have `HtmlUtils.append()` and `HtmlUtils.prepend()`?  I'm forgetting why this was preferred.